docs/network protocol.md

```diff
@@ -674,8 +674,8 @@ Tags are represented as a list of strings, the common Client tags follow:
 ### DeathLink
 A special kind of Bounce packet that can be supported by any AP game. It targets the tag "DeathLink" and carries the following data:
 
-| Name | Type | Notes |
-| ---- | ---- | ---- |
-| time | float | Unix Time Stamp of time of death. |
-| cause | str | Optional. Text to explain the cause of death, ex. "Berserker was run over by a train." |
-| source | str | Name of the player who first died. Can be a slot name, but can also be a name from within a multiplayer game. |
+| Name | Type | Notes                                                                                                                                                  |
+| ---- | ---- |--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| time | float | Unix Time Stamp of time of death.                                                                                                                      |
+| cause | str | Optional. Text to explain the cause of death. When provided, or checked, this should contain the player name, ex. "Berserker was run over by a train." |
```

Comment:
I would suggest removing the 'or checked' here as to me i have no idea what this is meant to mean

Reply:
when your client checks the death link packet to see what it should display.